4_predict_plots.R

```diff
@@ -1970,6 +1970,20 @@ p4_plot_targets_list <- list(
                       select(PRMS_segid, Date, AP, BR, VR, CP, PD)
                     )
   ),
+  #Make maps showing the reaches that each data split covers
+  tar_target(
+    p4_shap_data_split_maps_png,
+    plot_data_splits(split = p4_shap_data_splits[[1]], 
+                     split_name = names(p4_shap_data_splits)[1],
+                     #using p2_TOT_lc_physio_attrs because it's smaller than the full table
+                     full_df = full_df <- distinct(p2_TOT_lc_physio_attrs, PRMS_segid),
```

Comment:
Suggesting this change in case of a typo. 

And is `full_df` expected to contain columns other than `PRMS_segid`? I see that `plot_data_splits` has another call to `distinct(full_df, PRMS_segid)` so just want to make sure I'm understanding the inputs.

```suggestion
                     full_df = distinct(p2_TOT_lc_physio_attrs, PRMS_segid),
```

Reply:
Thanks! That was a typo.

`full_df` only needs the `PRMS_segid`